docs/fast_build.md

```diff
@@ -1,4 +1,113 @@
-Optimizing a Tiltfile
+Expanding a Tiltfile
 =====================
 
-TODO(nick): Write this guide.
+This tutorial looks at a `Tiltfile` with build optimizations.
+We explain what they do, and why you would want to use them.
+
+In the previous tutorial, we introduced the `static_build()` function.
```

Comment:
Maybe link to the previous tutorial in case someone lands here and wants to back track?

Reply:
done!